### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gerard5595**
Hallo

### CODE REVIEW

**Comment by Ghislain1** [comment]
@gerard5595  COOL;  you can do pr

File: README.md [hunk 1]
```diff
@@ -125,3 +125,4 @@ The following channels are available for discussions, feedback, and support requ
       [karma-mocha-reporter](https://github.com/litixsoft/karma-mocha-reporter) and
       [ng-mocks](https://github.com/ike18t/ng-mocks)
 1.  [] End-to-end tests with Protractor,
+1.  [] Hallo
```
Comment (Ghislain1) [inline README.md:128]: remove it

**Review by Ghislain1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gerard5595) [reply README.md:128]: Okya

Comment (gerard5595) [reply README.md:128]: Done

**Pull request closed without merging**